omod/src/main/java/org/openmrs/module/rwandahivflowsheet/web/model/FormDataModel.java

```diff
@@ -432,21 +435,32 @@ public String getCurrentDrugNames(String drugSetName) {
 			if (o.getDrug() != null)
 				ret.append(o.getDrug().getName());
 			else
-				ret.append(o.getConcept().getBestName(Context.getLocale()).getName());
+				ret.append(o.getConcept().getDisplayString());
 			if (i.hasNext())
 				ret.append(" ");
 		}
 		return ret.toString();
 	}
 	
+	@SuppressWarnings("unchecked")
+	public List<DrugOrder> getDrugOrdersByPatient(Patient patient){
+		
+		OrderType drugOrderType = Context.getOrderService().getOrderTypeByUuid(OrderType.DRUG_ORDER_TYPE_UUID);
+
+		OrderSearchCriteria orderSearchCriteria = new OrderSearchCriteriaBuilder().setPatient(patient)
+				.setOrderTypes(Collections.singletonList(drugOrderType)).build();
+
+		return (List)Context.getOrderService().getOrders(orderSearchCriteria);
+	}
 	
 	public List<DrugOrder> getAllPatientDrugOrders(){
 		List<DrugOrder> ret = new ArrayList<DrugOrder>();
 		if (allPatientDrugOrders == null || allPatientDrugOrders.size() == 0){
-				List<DrugOrder> tmp  = Context.getOrderService().getDrugOrdersByPatient(this.getPatient());
+				List<DrugOrder> tmp  = getDrugOrdersByPatient(this.getPatient());
+				Context.getOrderService().getAllOrdersByPatient(patient);
```

Comment:
This line seems like it is dangling, and probably was supposed to be removed from a previous commit.